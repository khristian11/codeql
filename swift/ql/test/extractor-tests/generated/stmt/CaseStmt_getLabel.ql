import codeql.swift.elements
import TestUtils

from CaseStmt x, int index
where toBeTested(x)
select x, index, x.getLabel(index)
