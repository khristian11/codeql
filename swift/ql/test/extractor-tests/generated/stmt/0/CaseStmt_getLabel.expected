| statements.swift:23:5:25:3 | case ... | 0 | statements.swift:23:11:23:11 | let ... |
| statements.swift:29:5:31:3 | case ... | 0 | statements.swift:29:11:29:11 | ... is ... |
| statements.swift:31:5:33:3 | case ... | 0 | statements.swift:31:11:31:11 | let ... |
| statements.swift:56:4:58:7 | case ... | 0 | statements.swift:56:9:56:9 | 1 |
| statements.swift:59:4:61:7 | case ... | 0 | statements.swift:59:9:59:9 | 5 |
| statements.swift:59:4:61:7 | case ... | 1 | statements.swift:59:12:59:12 | 10 |
| statements.swift:62:4:63:22 | case ... | 0 | statements.swift:62:4:62:4 | _ |
